showcase/assets/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+  "SHOWCASE": {
+    "HOME": {
+      "TITLE": "Stark Starter"
```

Comment:
Shouldn't we call this one "Showcase" instead of "Starter"

Reply:
I'll adapt the PR and also take the occasion to rebase